docs/buildspecs/compiler.rst

```diff
@@ -188,11 +188,11 @@ We can build this test by running::
 
     buildtest build -b $BUILDTEST_ROOT/examples/compilers/envvar_override.yml
 
-Next, let's see the generated test by running ``buildtest inspect query -d all -t override_environmentvars``. The
+Next, let's see the generated test by running ``buildtest inspect query -d all --testpath override_environmentvars``. The
```

Comment:
okay there is a typo the `-d all` is no longer valid option as part of `buildtest inspect query`. This option was deleted can you please remove.

Reply:
Yes, I'll remove it